store/tx_manager.go

```diff
@@ -475,39 +475,29 @@ func (txm *EthTxManager) handleUnconfirmed(
 	txat *models.TxAttempt,
 	blkNum uint64,
 ) (*TxReceipt, error) {
-	bumpable := tx.Hash == txat.Hash
-	pastThreshold := blkNum >= txat.SentAt+txm.config.EthGasBumpThreshold()
-	logger.Debugw(
-		fmt.Sprintf("TxManager handleUnconfirmed: tx attempt %s", txat.Hash.Hex()),
-		"currentBlockNumber", blkNum,
+	// Only handle the latest transaction attempt, not the remainder
+	if tx.Hash != txat.Hash {
```

Comment:
Possibly wrap this logic with a descriptively named method like `tx.IsLatestAttempt(txat)` instead of the comment.

Reply:
My only concern with that is that it's ambiguous, because the function can't really tell that a transaction indicates the latest attempt, only that two transactions are the same.

Reply:
I think I have something that'll do the trick right here https://github.com/smartcontractkit/chainlink/pull/985/files